pkg/updater/updater.go

```diff
@@ -175,16 +131,14 @@ func New(config Config, centralEndpoint string, db database.Datastore, repoToCPE
 		return nil, errors.Wrap(err, "getting relevant download URL")
 	}
 
-	client := &http.Client{
-		Timeout:   defaultTimeout,
-		Transport: proxy.RoundTripper(),
-	}
+	client := newHTTPClient(proxy.RoundTripper())
 	clientConfig, err := mtls.TLSClientConfigForCentral()
 	if err != nil {
 		return nil, errors.Wrap(err, "generating TLS client config for Central")
 	}
 	client.Transport = &http.Transport{
-		TLSClientConfig: clientConfig,
+		TLSClientConfig:    clientConfig,
+		DisableCompression: true,
```

Comment:
👍

Reply:
Can you leave a comment here why we do this? It may not be clear why we explicitly disable compression

Reply:
+1